src/Screens/DynamicForm/components/checkbox.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Form, Checkbox } from 'antd';
+
+export default function({ field }) {
+  return (
+    <Form.Item
+      label={field.key}
+      key={field.key}
+      help={field.help_info || ''}
+      required={field.required}
+    >
+      <Checkbox value={field.placeholder || null}/>
```

Comment:
Have you verified that this (using `value` prop) works as expected?

This is how I did for the TOS agreement checkbox.
https://github.com/careerfairsystems/nexpo-web/blob/e6366f1aab42f64a3a6c5c2112f933512ee760fa/src/Screens/DynamicForm/index.js#L57